config/initializers/room_type_plugin.rb

```diff
@@ -0,0 +1,6 @@
+PLUGINS = {
```

Comment:
Ok, we declared a constant that contains list of available room types. 
And, actually, we can extend this list. Good step. 

Now let think further (step back). 
What is semantic meaning of the PLUGINS?  It does not reflect its purpose - listing of available room types. What if we call it AVAILABLE_ROOM_TYPES

Now, class describing room can be named more naturally. e.g. SingleRoom, DoubleRoom, TrippleRoom, Halflux(Room). If we need to protect global namespace and avoid collisions, we could add a namespace like: RoomType::SingleRoom, RoomType::DoubleRoom, but unless we need it we can keep it simple, dry.

Now, PLUGINS (AVAILABLE_ROOM_TYPES) constant is declared in the initializer, place which is hard to guess and find.

What if we simply declare RoomType namespace or class, and have AVAILABLE_ROOMS there.

Then, registering additional room would look like `RoomType::AVAILABLE_ROOMS << LuxRoom`
Question: what is the issue of using `<<` here and how can we enhance it?